classes/Hook/HookDisplayHeader.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\Ps_Googleanalytics\Hooks;
+
+class HookDisplayHeader implements HookInterface
+{
+    private $module;
+    private $context;
+    private $params;
+    private $backOffice;
+
+    public function __construct(\Ps_Googleanalytics $module, \Context $context)
+    {
+        $this->module = $module;
+        $this->context = $context;
+    }
+
+    /**
+     * run
+     *
+     * @return void|string
+     */
+    public function run()
+    {
+        if (!\Configuration::get('GA_ACCOUNT_ID')) {
+            return;
+        }
+            $this->context->controller->addJs($this->module->getPathUri() . 'views/js/GoogleAnalyticActionLib.js');
```

Comment:
Something is broken here. The indentation is weird :thinking:

Reply:
Wow, I had to pull my branch to get this change. This is strange, the GithubAction must not pass with a such error !
Thanks for the review, I'm fixing that issue.